apps/users/schemas.py

```diff
@@ -139,38 +139,70 @@ class DeleteUserIn(BaseInSchema):
     user_id: int
 
 
-class StudentCheckExistanceIn(BaseInSchema):   # TODO spelling error
-    full_name: str
-    telephone_number: str
+class StudentCheckExistenceIn(BaseInSchema):
+    last_name: str = Field(default="Петренко", max_length=50)
+    first_name: str = Field(default="Петро", max_length=50)
+    telephone_number: str = Field(default="380979889988", max_length=12, min_length=12)
 
+    @validator('last_name')
+    def validate_last_name(cls, value):
+        if not value:
+            raise ValueError("The student's surname is mandatory!")
+        if not value.istitle():
+            raise ValueError("The first letter must be uppercase!")
+        return value
 
-class StudentCheckExistanceOut(BaseOutSchema):
+    @validator('first_name')
+    def validate_first_name(cls, value):
+        if not value:
+            raise ValueError("The student's name is mandatory!")
+        if not value.istitle():
+            raise ValueError("The first letter must be uppercase!")
+        return value
+
+    @validator('telephone_number')
+    def validate_telephone_number(cls, value):
+        if not value.isdigit():
+            raise ValueError('The phone number must consist of digits!')
+        return value
+
+
+class StudentCheckExistenceOut(BaseOutSchema):
     student: int
     token: str
     expires: datetime
 
 
 class CreateStudentIn(BaseInSchema):
-    full_name: str
-    telephone_number: str
+    last_name: str = Field(default="Петренко", max_length=50)
+    first_name: str = Field(default="Петро", max_length=50)
+    middle_name: str = None
+    telephone_number: str = Field(default="380979889988", max_length=12, min_length=12)
     course_id: int
     faculty_id: int
     speciality_id: int
     gender: str
 
-    @validator('full_name')
-    def validate_full_name(cls, value):
-        full_name = value.split()
-        if not full_name or len(full_name) < 2:
-            raise ValueError("The student's name and surname are mandatory!")
+    @validator('last_name')
+    def validate_last_name(cls, value):
+        if not value:
+            raise ValueError("The student's surname is mandatory!")
+        if not value.istitle():
+            raise ValueError("The first letter must be uppercase!")
+        return value
+
+    @validator('first_name')
+    def validate_first_name(cls, value):
```

Comment:
the same method we use in StudentCheckExistenceIn class, can we re-use it?

Reply:
ok